AxonDeepSeg/morphometrics/launch_morphometrics_computation.py

```diff
@@ -43,15 +52,166 @@ def launch_morphometrics_computation(path_img, path_prediction):
+
+            if args["sizepixel"] is not None:
+                psm = float(args["sizepixel"])
+            else: # Handle cases if no resolution is provided on the CLI
+
+                # Check if a pixel size file exists, if so read it.
+                if (current_path_target.parent / 'pixel_size_in_micrometer.txt').exists():
+
+                    resolution_file = open(current_path_target.parent / 'pixel_size_in_micrometer.txt', 'r')
+
+                    psm = float(resolution_file.read())
+                else:
+
+                    print("ERROR: No pixel size is provided, and there is no pixel_size_in_micrometer.txt file in image folder. ",
+                                    "Please provide a pixel size (using argument -s), or add a pixel_size_in_micrometer.txt file ",
+                                    "containing the pixel size value."
+                    )
+                    sys.exit(3)
+
+        
+            
+
+
+            x = np.array([], dtype=[
+                                        ('x0', 'f4'),
+                                        ('y0', 'f4'),
+                                        ('gratio','f4'),
+                                        ('axon_area','f4'),
+                                        ('myelin_area','f4'),
+                                        ('axon_diam','f4'),
+                                        ('myelin_thickness','f4'),
+                                        ('axonmyelin_area','f4'),
+                                        ('solidity','f4'),
+                                        ('eccentricity','f4'),
+                                        ('orientation','f4')
+                                    ]
+                            )
+            
+            # Compute statistics
+            stats_array = get_axon_morphometrics(im_axon=pred_axon, im_myelin=pred_myelin, pixel_size=psm, axon_shape=axon_shape)
+
+            for stats in stats_array:
+
+                x = np.append(x,
+                    np.array(
+                        [(
+                        stats['x0'],
+                        stats['y0'],
+                        stats['gratio'],
+                        stats['axon_area'],
+                        stats['myelin_area'],
+                        stats['axon_diam'],
+                        stats['myelin_thickness'],
+                        stats['axonmyelin_area'],
+                        stats['solidity'],
+                        stats['eccentricity'],
+                        stats['orientation']
+                        )],
+                        dtype=x.dtype)
+                    )
+
+
+
+            # save the current contents in the file
+            if not (filename.lower().endswith((".xlsx", ".csv"))):  # If the user didn't add the extension, add it here
+                filename = filename + ".xlsx"
+            try:
+                # Export to excel
+                pd.DataFrame(x).to_excel(current_path_target.parent /  filename)
```

Comment:
There is no export to csv file. A `.csv` extension in `-f` will results in an error `ValueError: No engine for filetype: 'csv' `

Reply:
Good catch, I'll update the code and tests for the same.

Reply:
Added test for saving filename with `.csv` extension and fixed that error.